website/src/pages/index.js

```diff
@@ -0,0 +1,153 @@
+        <div className="container">
+          <div className={styles.title}>
+            <img src="img/redux_white.svg" alt="Redux logo" width="100" height="100" />
+            <h1 className={`${styles.projectTitle} hero__title`}>{siteConfig.title}</h1>
+          </div>
+          <p className="hero__subtitle">{siteConfig.tagline}</p>
+          <div className={styles.buttons}>
+            <Link
+              className={classnames(
+                'button button--secondary button--lg',
+                styles.getStarted,
+              )}
+              to={useBaseUrl('introduction/quick-start')}
+            >
+              Get Started
+            </Link>
+          </div>
+        </div>
+      </header>
+      <main>
+        {features && features.length && (
+          <section className={[styles.features, styles.lightBackground].join(' ')}>
+            <div className={`container ${styles.featureBlock}`}>
+              <div className="row">
+                {features.map(({ image, title, content }, idx) => (
+                  <div
+                    key={idx}
+                    className={classnames('col col--3', styles.feature)}
+                  >
+                    {image && (
+                      <div className={`text--center ${styles.blockImage}`}>
+                        <img
+                          className={styles.featureImage}
+                          src={useBaseUrl(image)}
+                          alt={title}
+                        />
+                      </div>
+                    )}
+                    <h2 className={`text--center ${styles.featureTitle}`}>{title}</h2>
+                    {content}
+                  </div>
+                ))}
+              </div>
+            </div>
+          </section>
+        )}
+        {otherLiraries && otherLiraries.length && (
+          <section className={styles.features}>
+            <div className="container">
+              <div className="row">
+                <div className="col">
+                  <h2 className={`text--center ${styles.secondTitle}`}>Other Libraries from the Redux Team</h2>
+                </div>
+              </div>
+              <div className="row">
+                {otherLiraries.map(({ image, title, content, link }, idx) => (
+                  <div key={idx} className={classnames('col col--6', styles.feature)}>
+                    <h2 className="text--center">
+                      <a href={link}>
+                        {title}
+                        <img className={styles.libLinkImg} src={image} />
+                      </a>
+                    </h2>
+                    <p className="text--center">{content}</p>
+                  </div> 
+                ))}
+              </div>
+            </div>
+          </section>
+        )}
+      </main>
+    </Layout>
+  );
+}
+
+function docUrl(doc, language) {
+  return `${siteConfig.baseUrl}${language ? `${language}/` : ''}${doc}`
```

Comment:
is this even used ? there's no language here. It would be nice to understand whats happening on the source code